### ISSUE TITLE
Plot complex functions

### ISSUE THREAD

**Issue by GarkGarcia** [Mathics3/mathics-django#1]
This is a follow up to https://github.com/mathics/Mathics/issues/794:

> When using Plot on a function with an imaginary part (e.g. `Plot[Ix,{x,1,10}]`, only an empty coordinate system is returned. WolframAlpha's solution is to show two grafs: Plot[{Re[Ix],Re[Ix]},{x,1,10}]`, which might be a better default...

**Comment by mmatera** [comment]
This is true for Wolfram alpha, but not for WL, at least until v 11.3.0 :

>> Plot[Sin[x+2 I x],{x,-1,1}]

![imagen](https://user-images.githubusercontent.com/2862288/103153477-8ef07f80-476f-11eb-9912-6a191dc9f724.png)

**Comment by GarkGarcia** [comment]
> This is true for Wolfram alpha, but not for WL, at least until v 11.3.0 :
> 
> > > Plot[Sin[x+2 I x],{x,-1,1}]
> 
> ![imagen](https://user-images.githubusercontent.com/2862288/103153477-8ef07f80-476f-11eb-9912-6a191dc9f724.png)

Does that mean we're complying to "standard" bahaviour (the one exhibited by Mathematica)?
